app/workers/zomato_data_grabbing_worker.rb

```diff
@@ -0,0 +1,73 @@
+require 'json'
+class ZomatoDataGrabbingWorker
+  include Sidekiq::Worker
+  include Faraday
+
+  def perform(*args)
+    puts 'Start Downloading Data From Zomato...'
+
+    conn = prepare_connection
```

Comment:
You can keep `conn` as a class variable and initialize it in the constructor instead of passing it all around